bin/publish

```diff
@@ -64,6 +65,12 @@ function check-npm-dependencies
   console.log!
 
 
+function get-exocom-version
```

Comment:
This is an asynchronous function that's being called synchronously - we're waiting for the 'ended' event before we returning the result. On second thought, let's just use `child_process.exec-sync` (the synchronous version of `exec`: https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options. Especially since we don't care about monitoring the logs of this process.

Sorry about all the confusion with `ObservableProcess`. Feel free to use whatever library you see fit, we use `ObservableProcess` when we need to monitor (observe) the output of a process, like when waiting for certain strings to be logged. By no means it is the only way to run processes. Since we only care about the output here we can use `exec-sync`.